sample/allofhands_hawaii2016/session_types

```diff
@@ -0,0 +1,73 @@
+[
+
+        {   
+            "id": 1, 
+            "length": null, 
+            "name": "Welcome Reception"
+        },
+        {
+            "id": 2, 
+            "length": null, 
+            "name": "Breakfast"
+        }, 
+        {
+            "id": 3, 
+            "length": null, 
+            "name": "Lunch"
+        },
+        {
+            "id": 4, 
+            "length": null, 
+            "name": "Team Time"
+        }, 
+        {
+            "id": 5, 
+            "length": null, 
+            "name": "Team Time"
```

Comment:
@mahikaw i meant you have multiple session types with same name? what is the difference with session type of id 4 and id 5. This is where length comes in where we have one with 30 mins and the other of 1 hour.